tests/integration/test_scmsteprun_update_view.py

```diff
@@ -29,53 +38,16 @@ def test_partial_update_invalid_status(self, client, logged_in_user, scm_step_ru
         assert response.status_code == 400
         assert response.json()["status"][0] == '"bla" is not a valid choice.'
 
```

Comment:
below: since we just pass the data to a remote endpoint, and checking/setting end date is done there, we can remove some tests here